### PR TITLE
Meenu/fix: updated help center for eu

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — DISMISSED** [review]
LGTM

File: src/pages/help-centre/data/_trading.ts [hunk 1]
```diff
@@ -163,6 +167,7 @@ const trading: TQuestionsData = {
                 {
                     translation_text:
                         '_t_Build your trading bot for free on <0>DBot</0> or <1>Binary Bot</1>; no coding is needed. You’ll also find free pre-built strategies on DBot that you can customise to your needs._t_',
+                    eu_translation_text: '_t_ _t_',
```
Comment (mitra-deriv) [inline src/pages/help-centre/data/_trading.ts:170]: Just a question why we are using empty text if we don’t need it ? 🤔

**Review by NikitK-deriv — DISMISSED** [review]
LGTM, Please resolve merge conflicts 🙏

File: src/pages/help-centre/data/_account.ts [hunk 2]
```diff
@@ -398,6 +421,7 @@ const account: TQuestionsData = {
                         { key: 0, type: 'link', to: '/contact_us/?is_livechat_open=true' },
                         { key: 1, type: 'link', to: `${deriv_app_url}/account/self-exclusion` },
                     ],
+                    eu_translation_text: '_t_ _t_',
```
Comment (habib-deriv) [inline src/pages/help-centre/data/_account.ts:424]:
```suggestion
                    eu_translation_text: '',
```

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**